frontend/src/pages/member/LoginPage.vue

```diff
@@ -73,12 +73,35 @@ export default {
         return;
       }
       try {
-        // TODO login API를 작성해주세요.
-        // const { email, password } = this.member;
-        // const data = await fetch("/login")
-        // TODO member 데이터를 불러와 주세요.
-        // const member = wait fetch("/members/me")
-        // this.setMember(member);
+        const jsonData = {
+          email: this.member.email,
+          password: this.member.password
+        };
+
+        const option = {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json'
+          },
+          body: JSON.stringify(jsonData)
+        };
+
+        const response = await fetch("http://localhost:8080/login/token", option);
```

Comment:
`http://localhost:8080` 같은경우 환경에 따라 달라질 수 있기 때문에 보통 Path로만 요청하고 있어요 :) 
```suggestion
        const response = await fetch("/login/token", option);
```

Reply:
view.config의 수정을 통해, path로 요청할 수 있게 번경했습니다. 

```java
const path = require('path');

module.exports = {
  outputDir: path.resolve(__dirname, '../src/main/resources/static/'),
  devServer: {
    proxy: {
      '/api': {
        target: 'http://localhost:8080/',
        ws: true,
        changeOrigin: true,
      },
    },
    historyApiFallback: true,
  },

  transpileDependencies: [
    'vuetify'
  ]
}
```

또한 기본 path에 /api를 추가했어요.

72d7456